### DISCUSSION TITLE
How to set notifications for discussions

### DISCUSSION THREAD

**Discussion by jepsar** [primefaces/community#24] [PrimeFaces] [answered]
I would like to only get notifications from PrimeFaces (category) discussions. If that's not possible, I'd like to unsubscribe from all discussions. I can't find an option to do so. Did I miss something?

**Comment by cagataycivici** [comment]
I think this is a question better suited for;

https://github.com/orgs/community/discussions/categories/discussions

**Reply by jepsar** [reply]
If there would be a discussion per repository instead, then this problem would not exist.

**Reply by cagataycivici** [reply]
But then things will be messy and way too distributed due to number of projects.

**Reply by atkntepe** [reply]
Hi, I understand you have an issue, but we had to decide for not only Primefaces but all our projects and products. We had many pros and cons, but this was the best way for us. We wanted to avoid carrying the old forum's issues here. That's why it is a much simpler structure and easy to monitor and provide help. For notifications, if I find something, I'll let you know as well.

**Reply by SoyDiego** [reply]
Changing the forum was necessary and Github Discussions and Discourse are better alternatives to the old forum. But if we compare GitHub Discussions VS Discourse, Discourse is much better.
Here everything is not so organized, and Discourse would have been a better option (in my opinion) both for the user experience and for all the maintainers and collaborators of the projects. In addition to SEO.
Hopefully Primetek can reconsider Discourse.

Anyway, as I said at the beginning, changing the forum was necessary and we have improved, but we could have improved even more. In any case, you have my support and if I can collaborate or help with something, you can tell me and I will be happy to do so.

Thank you

**Reply by jepsar** [reply]
The big plus for GitHub is that all the contributors are here already.

**Reply by jepsar** [reply]
FYI "Create issue from discussion" would work if we had discussions per repo. I still think discussions per repo is the most sane thing to do.

**Reply by onursenture** [reply]
Actually, it's still possible with "Reference in new issue" option.

![image](https://github.com/primefaces/community/assets/244957/9bbd49ce-6693-42a9-9daf-e257819d714a)

**Comment by SoyDiego** [comment]
> I would like to only get notifications from PrimeFaces (category) discussions. If that's not possible, I'd like to unsubscribe from all discussions. I can't find an option to do so. Did I miss something?

Great question and I didn't find something to do it.
If you find something, please leave a comment here again :)

**Comment by onursenture** [comment, marked as answer]
![image](https://github.com/primefaces/community/assets/244957/70e5b27e-9c0a-47e7-bd1c-e9839d7727d7)

Have you tried the 'Ignore' option under 'Watch' in the [primefaces/community](https://github.com/primefaces/community)?

**Reply by jepsar** [reply]
Ha, I was not aware that the discussions were bound to that repository. It was indeed set to "All activity". Will set it to "Participation and @mentions".

**Reply by cagataycivici** [reply]
Nice, thanks Onur.
